ear-production-suite-plugins/plugins/direct_speakers/src/direct_speakers_frontend_connector.cpp

```diff
@@ -138,8 +138,12 @@ void DirectSpeakersJuceFrontendConnector::setSpeakerSetup(
     routingComboBoxLocked->clearEntries();
     auto layoutSizeFixed = layoutSize != 0 ? layoutSize - 1 : layoutSize;
     for (int i = 1; i + layoutSizeFixed <= 64; ++i) {
-      routingComboBoxLocked->addTextEntry(String(i) + String::fromUTF8("–") +
-                                          String(i + layoutSizeFixed));
+      if(layoutSize > 1) {
+        routingComboBoxLocked->addTextEntry(String(i) + String::fromUTF8("–") +
+                                            String(i + layoutSizeFixed));
+      } else {
+        routingComboBoxLocked->addTextEntry(String(i));
+      }
```

Comment:
This is a really small thing, but I find things a bit easier to read if the body of a conditional is as small as possible. In this case it's only the string added as a text entry that changes based on the index, so I'd start with
```c++
String routingDescription = String(i);
if(layoutSize > 1) {
  routingDescription += String::fromUTF8("-") + String(i + layoutSizeFized);
}
routingComboBoxLocked->addTextEntry(routingDescription);
```

I'd then be tempted to stick that into a function in an anonymous namespace.

```c++
String routingLayoutDescriptionAt(int position, int layoutSizeFixed);
```
so that all you get in the loop is
```c++
routingComboBoxLocked->addTextEntry(routingLayoutDescriptionAt(i, layoutSizeFixed));
```

It's nitpicking, and feels like overkill when it's something this simple but it makes it explicit what the entry depends on and (maybe?) makes it easier to read.